pheps/phep-0001.md

```diff
@@ -0,0 +1,366 @@
+
+* The PHEP editors review your PR for structure, formatting, and other errors. Approval criteria are:
+    * It is sound and complete. The ideas must make technical sense. The editors do not consider whether they seem likely to be accepted.
+    * It is of reasonable scope, not too broad or unfocused.
+    * The title accurately describes the content.
+    * The PHEP's language (spelling, grammar, sentence structure, etc.) should be correct and conformant. Markdown (MD) markup and other formatting should be clear and reasonable.
+
+    Editors are generally quite lenient about this initial review, expecting that problems will be corrected by the reviewing process. Correctness is the responsibility of authors and reviewers, not the editors.
+
+    If the PHEP isn't ready for approval, an editor will send it back to the author with specific instructions.
+
+* Once approved, they will assign your PHEP a number. All PHEPs are sequentially-numbered at time of assignment. "Special" numbers (X, 0, etc.) are not used. Upon approval, the author must mark the PHEP PR ready for review.
+
+The PHEP editors will not unreasonably deny approval of a PHEP. Reasons for denying PHEP status include duplication of effort, being technically unsound, not providing proper motivation or addressing backwards compatibility, or not in keeping with the PyHC philosophy.
+
+Standards Track PHEPs may consist of two parts, a design document and a reference implementation. It is generally recommended that at least a prototype implementation in an existing or new PyHC package be co-developed with the PHEP, as ideas that sound good in principle sometimes turn out to be impractical when implemented.
+
+## Discussing a PHEP
+<a name="discussing-a-phep"></a>
+Once the draft PHEP is submitted in an open PR and a PHEP number has been assigned, the PR discussion thread provides the central place to discuss and review its contents. The PHEP must be updated so that the `Discussions-To` header links to the PR. The discussion must also be announced by the PHEP authors via reasonable means; this includes (but is not necessarily limited to) the PyHC email list and the PyHC Matrix/Slack.
+
+Informal conversation may happen elsewhere, but the discussion of record is the public thread on the pull request. Substantive input into the PHEP or questions about it belong in the PR comments and side conversations should result in comments on the PR. This ensures everyone can follow and contribute, avoids fragmenting the discussion, and makes sure input is fully considered as part of the PHEP review process. Feedback on the PR thread is a critical part of what the community will consider when reviewing the PHEP. Individual comments or the GitHub "review" process may both be used. Commenters may use the GitHub "approve" feedback to indicate they believe the PHEP is ready for a vote; however, this does not change the PHEP status to "Final".
```

Comment:
This connects with two points that I think we've been struggling with throughout the process:
1. If the bar to clear is consensus, how much effort needs to go into collecting positive votes? Does it matter if somebody can't make the meeting to vote "yes" given that it's the objecting votes which have the most power? If we really care about capturing formal "yes" votes separate from the meeting, there's going to be more process.
2. What's the right balance between making the meeting a "privileged" place for discussion and decision, and allowing the participation of people who can't make the meeting (even virtually)? We could go with entirely online approvals and scratch a lot of the meeting, removing any quorum consideration.

Although I wanted to highlight those big issues, this wording might be resolvable without. I don't think we can use "approving reviews" as a formal vote without really making it formal--the review expires when new commits are pushed, and if we accept "expired" reviews there's the question of approval after the fact. I'd be inclined to say if we want these to be formal votes (to point 2 above), the whole process should be reworked and the PR should stay in draft status until the vote.

To better capture the spirit of the telecon, then:
'Commenters may use the GitHub "approve" feedback to indicate they support the PHEP; however, this does not make a formal vote or change the PHEP status to "Final".'

Reply:
[New wording](https://github.com/heliophysicsPy/standards/blob/68125caadd7c081ecc8fc9f803c6d3ebebebce58/pheps/phep-0001.md?plain=1#L111) which is basically identical to what I put in the comment above.

Reply:
Can the GitHub feedback method be used by someone to disapprove when they can't make one of the meetings? If so, then what effect should that have on the formal vote? Should that person be pursued for further conversation? But then we don't want the process hung when one person disapproves on GitHub but doesn't make themselves available for discussion.

Reply:
As written, a comment on the PR is required to disapprove. Agreed it may get lost in the conversation. I think there should be one mechanism for formal disapproval to minimize the chance of losing it. A different option would be to equire an email to the person holding the agenda for the meeting.

Regardless of how the objection is raised, if there are edits at the meeting, those in attendance can decide if they feel the edits are sufficient to address the concerns raised by the person not there. If they do approve after edits, there will still be at least a month with no edits before the final vote.